apps/site/components/EOL/Table.tsx

```diff
@@ -0,0 +1,62 @@
+import { getTranslations } from 'next-intl/server';
+import type { FC } from 'react';
+
+import FormattedTime from '#site/components/Common/FormattedTime';
+import DetailsButton from '#site/components/Downloads/DownloadReleasesTable/DetailsButton';
+import provideReleaseData from '#site/next-data/providers/releaseData';
+import provideVulnerabilities from '#site/next-data/providers/vulnerabilities';
+
+import VulnerabilityChips from './VulnerabilityChips';
+
+const EOLTable: FC = async () => {
+  const releaseData = provideReleaseData();
+  const vulnerabilities = await provideVulnerabilities();
+  const EOLReleases = releaseData.filter(
+    release => release.status === 'End-of-life'
+  );
+
+  const t = await getTranslations();
+
+  return (
+    <table id="tbVulnerabilities" className="download-table full-width">
```

Comment:
```suggestion
    <table id="tbVulnerabilities">
```

copypasta from the downloads table